features/FEATURE_BLE/source/generic/GenericGap.cpp

```diff
@@ -1782,7 +1782,8 @@ void GenericGap::on_disconnection_complete(const pal::GapDisconnectionCompleteEv
 void GenericGap::on_connection_parameter_request(const pal::GapRemoteConnectionParameterRequestEvent &e)
 {
     if (_user_manage_connection_parameter_requests) {
-        _eventHandler->onUpdateConnectionParametersRequest(
+        if (_eventHandler) {
```

Comment:
If the event handler is not set, then the connection will crash. I think it is fair as it is a programming error. 
@donatieng What do you think ?

Reply:
Yes, happy with that - that's much better than something failing silently

Reply:
will add, thanks